crypto/fipsmodule/ml_kem/ml_kem_ref/kem.c

```diff
@@ -48,19 +68,20 @@ int crypto_kem_keypair_derand(ml_kem_params *params,
 *              - uint8_t *sk: pointer to output private key
 *                (an already allocated array of KYBER_SECRETKEYBYTES bytes)
 *
-* Returns 0 (success)
+* Returns 0 on success, aborts on failure.
 **************************************************/
 int crypto_kem_keypair(ml_kem_params *params,
```

Comment:
Is it possible to add a death test in `crypto/evp_extra/evp_extra_test.cc` as done in https://github.com/aws/aws-lc/blob/25709caeecb4f8206afc75bfd931599cf741b8dd/crypto/fipsmodule/ec/ec_test.cc#L1493

Reply:
not really because the PCT is part of keygen itself (and KEM params are hard-coded constants). In a future change I think I'll pull out the PCT as a separate function and add it as one of KEM methods and then call it in the high-level API. Maybe then we'll be able to implement a death test.